aws/service.go

```diff
@@ -1763,6 +1764,15 @@ func getSessionWithMaxRetries(ctx context.Context, d *plugin.QueryData, region s
 	// get aws config info
 	awsConfig := GetConfig(d.Connection)
 
+	// Configure the maximum number number of retry user wants to.
+	if awsConfig.MaxRetryCount != nil && *awsConfig.MaxRetryCount != "" {
+		maxRetryValue, err := strconv.Atoi(*awsConfig.MaxRetryCount)
+		if err != nil{
+			panic("Provide an Integer value in spc file for retry count")
```

Comment:
This doesn't seem to be the pattern we use when other config args are incorrect/not set (for example, see below). Where did you find this pattern?